fixham/pkg/api/calicobuilder.go

```diff
@@ -0,0 +1,18 @@
+package api
+
+// CalicoBuilder is a component in the Calico project
+type CalicoBuilder struct {
```

Comment:
I'm not sure I understand the point of this structure any more. It's a builder that overrides the Path() implementation, but why?

Reply:
This might be another one that isn't needed any more in the newer factoring of this code?